app/controllers/api/functions.php

```diff
@@ -442,6 +448,25 @@
         ]), Response::MODEL_RUNTIME_LIST);
     });
 
+App::get('/v1/functions/specs')
```

Comment:
lets not use short forms and use `specifications` or `sizes` since the param is called `size`